hacspec/src/math_integers.rs

```diff
@@ -505,44 +538,43 @@ macro_rules! signed_integer {
             #[cfg_attr(feature = "use_attributes", library(hacspec))]
```

Comment:
Hum didn't catch it earlier but this can't be `library`, we don't want to formalize strings. It should be switched to `primitive`.

Reply:
Ah, good catch. I"ll fix this everywhere